frontend/apps/reader/modules/readergesture.lua

```diff
@@ -191,6 +191,12 @@ function ReaderGesture:init()
         one_finger_swipe_top_edge_left = "ignore",
         one_finger_swipe_bottom_edge_right = "ignore",
         one_finger_swipe_bottom_edge_left = "ignore",
+        double_tap_left_side = self.ges_mode == "gesture_reader" and "prev_chapter",
+        double_tap_right_side = self.ges_mode == "gesture_reader" and "next_chapter",
```

Comment:
> Also replaces the default with +10/-10 pages instead of prev/next chapter.

Not yet done ?

Reply:
Whoops, I thought I'd picked one. I couldn't quite decide whether it was better to stick to the current or change it. Thanks for being vigilant. :-)

(Any thoughts?)

Reply:
(not really, I have double tap disabled - May be +10/-10 is better for skimming, some books might not have any chapter)

Reply:
Yeah, I don't use it either. As I said in the referenced issue I don't like the delay, even when I make it about a 100. Much lower than that isn't practical.